### PR TITLE
feat(org-stats): Add UsageChart base component

### PR DESCRIPTION
**Pull request by leedongwei**

### CODE REVIEW

File: src/sentry/static/sentry/app/views/usageStats/usageStatsOrg.tsx [hunk 1]
```diff
@@ -1,91 +1,152 @@
...
     ];
   }
 
+  get chartMetadata() {
+    const {orgStats} = this.state;
+
+    return {
+      ...this.mapStatsToChart(orgStats),
+    };
+  }
```
Comment (leedongwei) [inline src/sentry/static/sentry/app/views/usageStats/usageStatsOrg.tsx:82]: This method may return more keys when `getEndpoints` return something is made, so I'm putting it in first.

File: src/sentry/static/sentry/app/views/usageStats/usageChart/index.tsx [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+  ACCEPTED = 'Accepted',
+  DROPPED = 'Dropped',
+  PROJECTED = 'Projected',
+}
+
+type Props = {
+  theme: Theme;
+
+  title?: React.ReactNode;
+  footer?: React.ReactNode;
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/usageStats/usageChart/index.tsx:79]: I like this separation of footer and passing the active selection state in as props :clap:

File: src/sentry/static/sentry/app/views/usageStats/usageChart/index.tsx [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+    }
+
+    const chartData: ChartStats = {
+      accepted: [],
+      dropped: [],
+      projected: [],
+    };
+
+    const isCumulative =
+      !skipDataTransform && dataTransform === ChartDataTransform.CUMULATIVE;
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/usageStats/usageChart/index.tsx:164]: Would it be more extensible if the data transform was passed as a function? with the total, new value, and dataTransform as parameters? The default prop could do the cumulative / day-day value.

Alternatively should the conversion of stats -> series be something that a parent can replace? It seems like we have similar logic to this already in getsentry.

**Review by markstory — APPROVED** [review]
UsageChart changes look good to me.

**Pull request merged**